pennylane/templates/subroutines/trotter.py

```diff
@@ -126,7 +126,11 @@ def my_circ():
     .. details::
         :title: Usage Details
 
-        One can recover the behaviour of :class:`~.ApproxTimeEvolution` by setting :code:`order=1`.
+        This operation is similar to the :class:`~.ApproxTimeEvolution`. One can recover the behaviour of
+        :class:`~.TrotterProduct` by simply taking the adjoint:
+
+        >>> qml.adjoint(qml.ApproxTimeEvolution(hamiltonian, time, n=n))  # for order = 1
```

Comment:
```suggestion
        >>> qml.adjoint(qml.ApproxTimeEvolution(hamiltonian, time, order = 1, n=n))
```
Minor point though, as the default is one.

Reply:
`qml.ApproxTimeEvolution` is supposed to have order?

Reply:
No it doesn't have an order argument

Reply:
The last sentence sounds strange to me. I suggest using the same sentence and example used for  `ApproxTimeEvolution`, i.e.:

```
One can recover the behaviour of :class:`~.ApproxTimeEvolution` by
 taking the adjoint:

>>> qml.adjoint(qml.TrotterProduct(hamiltonian, time, order=1, n=n))
```

The reason is  `TrotterProduct` is the general operator and `ApproxTimeEvolution` is a special case of it not the other way.

Reply:
It felt awkward to me aswell, but I convinced myself that we should reference the other class in the code example. Swapping back to your suggestion